templates/drop/config_drop_workflow.tmpl

```diff
@@ -14,7 +14,7 @@ paths:
 options:  
   loglevel: INFO
 
-# all the variable listed here will be processed by the LRA creator
+# all the variable listed here will be processed by the DROP user
```

Comment:
this is - at this stage - still a LRA call in the workflow. please revert the comment